README.md

```diff
@@ -1,52 +1,124 @@
 
-The ApostropheCMS documentation will walk you through [editing page templates](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/editing-page-templates), [global content](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/global), [creating custom widgets](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/custom-widgets) and [navigation](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/building-navigation), along with [configuring reusable "pieces"](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/reusable-content-with-pieces) and [global settings](https://docs.apostrophecms.org/apostrophe/core-concepts/global-settings/settings). Plus, a whole lot more!
+#### New Project
+Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli). Alternatively, you can simply clone this repo and `npm install`.
+##### Install the Apostrophe CLI
+ ```sh
+  npm install -g apostrophe-cli
+ ```
+##### Create a project
+ ```sh
+ apos create-project <shortname-without-spaces>
+ ```
 
-## Taking it Further
+`<shortname-without-spaces>` represents the "shortname" of the application. It will be used by default as a MongoDB database name and a basis for cookie names, etc. You will find this as the shortName property in app.js.
 
-Our documentation also covers advanced topics such as [managing permissions](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/permissions), [working with ApostropheCMS data](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/model-layer), [building forms](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/forms), and [hosting an ApostropheCMS site in production](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/deployment).
 
-Moreover, there's tons of great info in the ["How To"](https://docs.apostrophecms.org/apostrophe/tutorials/howtos) section of the docs, and it's always growing!
+## Extensions and Integrations
 
-## Extending the Core with Modules
+You can find a complete list of [extensions](https://apostrophecms.com/extensions) on our website, but here are a few highlights:
 
-ApostropheCMS is Node.js all the way through, which means it can be [extended](https://apostrophecms.com/extend) in all sorts of interesting ways. Here's a list of some of our favorite modules:
+  [apostrophe-headless](https://github.com/apostrophecms/apostrophe-headless) - Add REST APIs and power your React / Vue Native / etc. apps with a headless CMS.
+  [apostrophe-blog](https://github.com/apostrophecms/apostrophe-blog) - Everything you need to put a blog on your site.
+  [apostrophe-events](https://github.com/apostrophecms/apostrophe-events) - Manage upcoming events.
```

Comment:
"Manage one-time and recurring events." "Upcoming" isn't really important one way or the other.

Reply:
I realize this was from before.

Reply:
Sure. Upcoming-ness is a key reason why this module is needed, but that's not obvious enough that it should go in this sentence.

Reply:
True.